Packages/com.unity.inputsystem/InputSystem/Plugins/PlayerInput/PlayerInput.cs

```diff
@@ -693,7 +779,18 @@ public void ActivateInput()
                 m_CurrentActionMap?.Enable();
         }
 
-        public void PassivateInput()
```

Comment:
While this change is the right thing to do, it will break user's projects. Leave an obsolet `PassivateInput` method which calls the new one.

Reply:
Fixed